src/main/resources/com/dansplugins/factionsystem/db/migration/V7__Store_law_index.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `mf_law`;
+
+ALTER TABLE `mf_law` ADD `number` integer NOT NULL;
```

Comment:
Adding a default here or allowing null could prevent us having to empty the table.